plugins/edge/plugin.go

```diff
@@ -33,17 +33,22 @@ const (
 )
 
 type Config struct {
-	Enabled           bool          `json:"enabled"`
-	Addr              string        `json:"addr"`
-	APIKey            string        `json:"apikey"`
-	TenantID          string        `json:"tenant_id,omitempty"`
-	Timeout           int           `json:"timeout"`
-	SyncInterval      int           `json:"sync_interval"`
-	Insecure          bool          `json:"insecure"`
-	SessionID         string        `json:"session_id,omitempty"`
-	ConnectionTimeout time.Duration `json:"-"`
-	// Deprecated: No longer used.
-	PageSize int `json:"page_size,omitempty"`
+	Enabled           bool              `json:"enabled"`              //
```

Comment:
Would it make sense to embed a go-aserto `client.Config` struct instead of duplicating the fields?

Reply:
I do not think it is possible as the json tags are not aligned

```
addr vs address
apikey vs api_key
```

Reply:
Makes sense. We should probably do it as part of v33 though.

Reply:
I can do it in my config refactoring branch.